src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -162,6 +165,22 @@ namespace winrt::TerminalApp::implementation
         _filteredActionsView().ScrollIntoView(_filteredActionsView().SelectedItem());
     }
 
+    winrt::Microsoft::Terminal::TerminalControl::KeyChord CommandPalette::SetKeyChord(const bool ctrl, const bool alt, const bool shift, const winrt::Windows::System::VirtualKey key)
+    {
+        winrt::Microsoft::Terminal::TerminalControl::KeyChord kc{};
+
+        for (int vKey = 0; vKey != 219; vKey++)
+        {
+            VirtualKey virtualKey = static_cast<VirtualKey>(vKey);
+            if (virtualKey == key)
+            {
+                kc = { ctrl, alt, shift, static_cast<int32_t>(virtualKey) };
+                return kc;
+            }
+        }
+        return kc = nullptr;
```

Comment:
Why isn't this whole function just 
```suggestion
        return winrt::Microsoft::Terminal::TerminalControl::KeyChord{ctrl, alt, shift, static_cast<int32_t>(key)};
```